### PR TITLE
Add pep8 master

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by aashish24** [comment]
#1289 #1290

File: CMake/cdat_modules/pep8_pkg.cmake [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+set( PEP8_URL ${LLNL_URL} )
+set( PEP8_GZ pep8-${PEP8_VERSION}.tar.gz )
+set( PEP8_MD5 c7a3f57d832484a6295164661fbb1335 )
+
+set (nm PEP8)
+string(TOUPPER ${nm} uc_nm)
+set(${uc_nm}_VERSION ${${nm}_MAJOR}.${${nm}_MINOR}.${${nm}_PATCH})
+set(PEP8_SOURCE ${PEP8_URL}/${PEP8_GZ})
+
+if (BUILD_TESTING)
```
Comment (doutriaux1) [inline CMake/cdat_modules/pep8_pkg.cmake]: do we turn on build testing? is it on by default?

Comment (aashish24) [inline CMake/cdat_modules/pep8_pkg.cmake]: It is on by default.

Comment (aashish24) [inline CMake/cdat_modules/pep8_pkg.cmake]: We can turn it OFF by default but I think its mostly us (developers) building it for now.

Comment (doutriaux1) [inline CMake/cdat_modules/pep8_pkg.cmake]: if we turn it off, does that means ctest will not work? If so, leaving it ON is probably best.

Comment (aashish24) [inline CMake/cdat_modules/pep8_pkg.cmake]: Right, there won't be any ctesting if it is OFF. I agree with you having testing ON by default is good. Having PEP installed when testing is ON is good IMO. If you agree, we can have it merged and then @dlonie can add style testing.

**Comment by aashish24** [comment]
I am merging this one since this wasn't really my  branch and I approved it

**Pull request merged**